README.md

```diff
@@ -296,7 +296,12 @@ You can delete the <writable_image_dir> now.
 
 ## 2. Configuration
 
-Adjust `config/config.yaml` for your particular use case.
+Adjust `config/config.yaml` for your particular use case. It is possible to use pre-calculated features (e.g. [downloaded from our features database](https://github.com/KosinskiLab/AlphaPulldown?tab=readme-ov-file#installation)) by adding paths to the features to your config/config.yaml
+
+```yaml
+feature_directory :
+  - "/path/to/directory/with/features/"
```

Comment:
From my tests, currently, this is not sufficient, one has to also set `--compress_features: ` to True if the folders contain compress features. Do I see it correctly? This either needs to be explained in the documentation or preferably fixed in the pipeline so that it checks for both compressed and uncompressed version.

Reply:
Added !Note to docs, making it work for both compressed and uncompressed is on our to-do list with Valentin, but not so simple technically.